_topics/en/unannounced-channels.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Unannounced channels
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Private channels
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Unannounced channels** are LN channels that are not advertised to
+  the network for use in routing.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Most unannounced channels are believed to belong to users that simply
+  don't intend to route payments, such as users of mobile clients that
+  aren't always online to route payments.
+
+  Unannounced channels channels are sometimes called **private
+  channels** but this is [discouraged][unpublished channels delenda est]
+  by some experts because, if the channels aren't used for routing, then
+  the publicly known counterparty of an unannounced channel knows that
+  any payment to or from that channel either started or ended with that
+  channel.
+
+  [unpublished channels delenda est]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-December/002408.html
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "LND #1944 tweaks `sendtoroute` RPC to allow routing via private channels"
+    url: /en/newsletters/2018/11/13/#lnd-1944
+    date: 2018-11-13
+
+  - title: "C-Lightning #2230 adds a `private` flag to the `listpeers` RPC"
+    url: /en/newsletters/2019/01/15/#c-lightning-2230
+    date: 2019-01-15
+
+  - title: "C-Lightning #2234 adds route hints for private channels to `invoice` RPC"
+    url: /en/newsletters/2019/01/22/#c-lightning-2234
+    date: 2019-01-22
+
+  - title: "Talk about LN topology, private channels not included"
```

Comment:
"Talk about LN topology, private channels not included" is a bit confusing because they're explicitly mentioned in the talk and write-up.